setup_environment.md

```diff
@@ -0,0 +1,45 @@
+# Set up the conda environment for STEMMUS-SCOPE
+
+In this guide, you will learn how to configure the conda environment in your jupyter-workspace on SURF research cloud for running the STEMMUS-SCOPE model in the [notebook](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/docs/notebooks/run_model_in_notebook.ipynb).
+
+This guide assumes you already have access to SRC as a user and the workspace created by the administrator of EcoExtreML project group.
+
+In order to run STEMMUS-SCOPE on SRC, we need to configure the `conda` environment and download the executable file.
+
+## Managing environments and kernels on this Jupyter-workspace using `conda`
+In our jupyter-workspace, `miniconda` is pre-installed at the location `/etc/miniconda`. We will use it to create virtual python environments and install [`pystemmusscope`](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing).
```

Comment:
/etc is a weird location for software. /etc is usually for configuration files. Did you install it there or is this part of the chosen application?

Reply:
No, I didn't. This was pre-installed when Sarah and I generate the environment this `conda` thing is already there. It seems this is something new (or maybe hidden before) by SURF.